Longest_Common_Subsequence/Longest_Common_Subsequence.cpp

```diff
@@ -0,0 +1,38 @@
+/* Problem Statement- Given two sequences, find the length of longest subsequence present in both of them. A subsequence is a sequence that can be derived from another sequence by deleting some or no elements without changing the order of the remaining elements.*/
+#include<bits/stdc++.h> 
+using namespace std;
+
+// Function to calculate length of longest common subsequence
+int LongestCommonSubsequence(int arr1[], int arr2[], int n, int m) 
+{ 
+   int LCS[n+1][m+1];
+   for (int i=0; i<=n; i++) 
+   { 
+        for (int j=0; j<=m; j++) 
+        { 
+            if (i == 0 || j == 0) 
+                LCS[i][j] = 0; 
+            else if (arr1[i-1] == arr2[j-1]) 
```

Comment:
Please add some comment about the logic. It would be great.

Reply:
added explanation